papers/ring_vrf/abstract-short.tex

```diff
@@ -1,35 +1,35 @@
 
 \def\eprintsmallskip{\smallskip}{}%
 We introduce a new cryptographic primitive, aptly named
- \emph{ring verifiable random functions (ring VRF)},
- as a broadly superior approach to anonymous credentials.
+\emph{ring verifiable random functions (ring VRF)}.
```

Comment:
Agreed.

Reply:
plurality is not consistent: either "function" or "ring VRFs"

Reply:
I vote for the former